<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendItem</class>
 <widget class="QWidget" name="friendItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>73</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>73</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>73</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/tx/24.png</normaloff>:/tx/24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>麻辣大西瓜</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_feeling">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>好人一生平安</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
